src/Components/PostCard.js

```diff
@@ -1,5 +1,6 @@
 import React, { useState } from "react";
 import { useDispatch, useSelector } from "react-redux";
+import { handleBookmark, handleUnBookmark } from "../features/auth/authSlice";
```

Comment:
can use absolute imports, you can refer this 
https://dev.to/mr_frontend/absolute-imports-in-create-react-app-3ge8